include/seahorn/HornClauseDB.hh

```diff
@@ -129,9 +132,18 @@ namespace seahorn
     /// resets all indexes
     void resetIndexes ();
 
+    /// callgraph
+    typedef std::map<Expr, expr_set_type > callgraph_type;
```

Comment:
I think the callgraph should be factored out of `HornClauseDB`. The database is just a collection of clauses. Structures that build on it should be defined separately on the side.